lib/active_merchant/billing/gateways/mastercard.rb

```diff
@@ -7,9 +7,20 @@ def initialize(options={})
       end
 
       def purchase(amount, payment_method, options={})
-        MultiResponse.run do |r|
-          r.process { authorize(amount, payment_method, options) }
-          r.process { capture(amount, r.authorization, options) }
+        if options[:pay_mode]
```

Comment:
Is this flag available for all the regional endpoints?

Reply:
Yep